static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -708,26 +766,20 @@ require.def(
                 throw "ApiError: " + errorMessage;
             },
 
-            _setDisplayFullScreenForVideo: function() {
```

Comment:
Ignore this, saw your comment around this not being required for full screen playback.